subiquity/server/controllers/install.py

```diff
@@ -380,6 +385,61 @@ async def run_curtin_step(name, stages, step_config, source=None):
                     step_config=self.rp_config(logs_dir, mp.p()),
                     source='cp:///cdrom',
                     )
+            await self.create_rp_boot_entry(context=context, rp=rp)
+
+    @with_context(description="creating boot entry for reset partition")
+    async def create_rp_boot_entry(self, context, rp):
+        fs_controller = self.app.controllers.Filesystem
+        if not fs_controller.reset_partition_only:
+            cp = await self.app.command_runner.run(
+                ['lsblk', '-n', '-o', 'UUID', rp.path],
+                capture=True)
+            uuid = cp.stdout.decode('ascii').strip()
+            conf = grub_reset_conf.replace("#UUID#", uuid)
+            conf = conf.replace("#PARTITION#", str(rp.number))
+            with open(self.tpath('etc/grub.d/99_reset'), 'w') as fp:
```

Comment:
I like the convention that files we generate on the target system get a header that helps the end user figure out how they got there.  Something akin to `generate_timestamped_header()` would be nice.